packages/scripts/src/tasks/providers/ios.ts

```diff
@@ -56,6 +76,43 @@ function convertToValidIOSAppId(appId: string) {
   return appId.replace(/_/g, "-");
 }
 
+/**
+ * Get deployment-specific custom URL schemes, used for deep linking.
+ * Currently just REVERSED_CLIENT_ID, required for Google Auth integration
+ * TODO: make this step optional, allowing for deployments without Google Auth integration, for example
+ */
+function getCustomUrlSchemes(authProvider: IiOSBuildOptions["authProvider"]) {
+  let GOOGLE_REVERSED_CLIENT_ID = "";
+  if (authProvider) {
+    // TODO: handle other auth providers
+    if (authProvider === "firebase") {
```

Comment:
nit(non-blocking): I'm not sure if this method really needs generalising right now as even if using supabase I'm pretty sure we'd need google services/plist if wanting to offer google-based sign-in, although if looking ahead I'd say might be good to throw an error if any other authProvider detected just so we know we need to review the code